lib/models/App.js

```diff
@@ -536,13 +597,16 @@ class App {
    */
 
   prepare() {
-
     this.loadConfig()
 
     this.context.Promise = Promise
     this.context.request = request
 
     this.loadFramework()
+    this.loadPlugins()
+
+    this.executePluginPhase('didLoadFramework')
```

Comment:
consider wrap the phasing callback with a function to automatically fire will/did/etc events
like 
callbackable('framework', () => {
  this.loadFramework()
  this.loadPlugins()  
})